### PR TITLE
feat: add disable 2fa modal

### PR DESCRIPTION
**Pull request by alexbarnsley**
## Summary

https://app.clickup.com/t/p5fgqb

## Checklist

- [ ] Documentation _(if necessary)_
- [ ] Tests _(if necessary)_
- [x] Ready to be merged

### CODE REVIEW

**Comment by ItsANameToo** [comment]
@alexbarnsley what's the difference between this and https://github.com/ArkEcosystem/laravel-fortify/pull/165

**Comment by alexbarnsley** [comment]
oh noes. I didn't see that. I guess they achieve the same thing so ultimately there's no difference. i guess you get the fun job of choosing which you prefer :hide:

**Comment by alexbarnsley** [comment]
@ItsANameToo I've done Jolan's points here FYI https://github.com/ArkEcosystem/laravel-fortify/pull/165#pullrequestreview-709315759

**Comment by Highjhacker** [comment]
Looking good to me @alexbarnsley, fixed the issues I mentioned in the previous PR. One thing tho, whenever I input my password to disable the 2FA, even a wrong password, I get the modal opened from chrome asking me if I want to update my password for the logged in account on that website, is there a way to get rid of this ? 🤔 It's pretty annoying and shouldn't be there since we don't update the password at all here.

**Comment by alexbarnsley** [comment]
will take a look @Highjhacker 🤔

**Comment by alexbarnsley** [comment]
Updated the password-toggle component here - a property to mark the field as masked which uses styling to hide the characters for a "text" input field: https://github.com/ArkEcosystem/laravel-ui/pull/577

**Comment by alexbarnsley** [comment]
@ItsANameToo @Highjhacker

**Comment by Highjhacker** [comment]
Will have a look now @alexbarnsley

**Review by Highjhacker — APPROVED** [review]

File: resources/views/components/modals/2fa-password-confirmation.blade.php [hunk 1]
```diff
@@ -0,0 +1,75 @@
+@props([
+    'actionMethod',
+    'closeMethod',
+    'title',
+    'description',
+    'image' => '/images/auth/confirm-password.svg',
```
Comment (ItsANameToo) [inline resources/views/components/modals/2fa-password-confirmation.blade.php]: does the project need to supply an image to use? if so, maybe useful for the readme? 🤔

Comment (alexbarnsley) [reply resources/views/components/modals/2fa-password-confirmation.blade.php]: it does indeed, I'll add it to the readme

**Pull request merged**